tests/functional/pyocf/types/cache.py

```diff
@@ -248,7 +248,10 @@ def __init__(
         self.cores = []
 
     def start_cache(
-        self, init_mngmt_queue=True, init_default_io_queue=True, locked: bool = False,
+        self,
+        init_mngmt_queue=True,
+        init_default_io_queue=True,
+        locked: bool = False,
```

Comment:
Major part of this commit is actually a refactoring, not really related to the commit title. Please extract the refactoring part to the separate commit.